frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -471,12 +486,11 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                         searchPlaceholder: 'exceptions',
                         type: TaxonomicFilterGroupType.ErrorTrackingProperties,
                         options: [
-                            ...[
-                                '$exception_types',
-                                '$exception_values',
-                                '$exception_sources',
-                                '$exception_functions',
-                            ].map((value) => ({ name: value, value, group: TaxonomicFilterGroupType.EventProperties })),
+                            ...getInternalEventPropertyFilterOptions('error-tracking').map((value) => ({
+                                name: value,
+                                value,
+                                group: TaxonomicFilterGroupType.EventProperties,
+                            })),
```

Comment:
This is confusing because "Exception properties" aren't always from internal events. We use this category to show a custom tab in the taxonomic filter on the error tracking list page so that people can search the properties that always exist on exception events
<img width="908" height="503" alt="Screenshot 2025-10-20 at 09 44 22" src="https://github.com/user-attachments/assets/346a17bf-7e33-4351-99cd-ec39afcd3716" />

Reply:
> I assumed we treated exceptions as an internal event type

It's a little confusing. The internal events are `$error_tracking_issue_created` and `$error_tracking_issue_reopened`. We do add the properties of the exception event that causes those internal events as properties of the internal event. But also treat the `$exception` as an ordinary event